### ISSUE TITLE
Structure of things_id.json, things_key.json,  channels_id.json  files.

### ISSUE THREAD

**Issue by sitewhere-iot** [mainflux/benchmark#9]
what does the structure of these files look like?

**Comment by drasko** [comment]
Indeed, @blokovi it is unclear [here](https://github.com/mainflux/benchmark/tree/master/mzbench#1-to-1) what these files should look like.

Please improve documentation by giving the examples of these files, or mentioning where these come from - do they come from [bulk provisioning](https://mainflux.readthedocs.io/en/latest/platform-management/#bulk-provisioning-things)? If yes - improve Bulk Provisioning documentation section and give the example of these JSON files, the explain here the relation.

**Comment by sitewhere-iot** [comment]
when will I get an answer?

**Comment by drasko** [comment]
When you remove 3 exclamation marks from the title - they are not needed.

**Comment by sitewhere-iot** [comment]
I removed it but there is still no response

**Comment by drasko** [comment]
Because you left a full-stop. Titles do not have interpunction signs.
